purchase_packaging_level_qty/models/purchase_order.py

```diff
@@ -26,9 +32,20 @@ def _compute_purchase_transport_packaging_level(self):
     def _compute_total_transport_packaging_qty(self):
         for order in self:
             if order.transport_packaging_level_id:
-                total_transport_qty = sum(order.mapped("order_line.transport_qty"))
-                order.total_transport_qty = "{} {}".format(
-                    total_transport_qty,
+                order.total_transport_qty = sum(
+                    order.mapped("order_line.transport_qty")
+                )
+            else:
+                order.total_transport_qty = False
+
+    @api.depends(
+        "total_transport_qty",
```

Comment:
```suggestion
        "total_transport_qty", "transport_packaging_level_id",
```

Reply:
I did not added dependency on  `transport_packaging_level_id` because `total_transport_qty ` is already dependent on it.
But add it here as well to be more explicit.